casanovo/casanovo.py

```
@@ -581,7 +512,7 @@ def setup_model(
     return config, model
 
 
-def _get_model_weights(cache_dir: Path) -> Path:
+def _get_model_weights(cache_dir: Path, is_timstof: bool = False) -> Path:
```

Ruff (0.15.10)

[warning] Boolean-typed positional argument in function definition

(FBT001)